composer.json

```diff
@@ -3,15 +3,18 @@
     "infinum/coding-standards-wp": "0.4.1",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.5"
   },
+  "require": {
+    "infinum/eightshift-libs": "dev-develop"
```

Comment:
This will have to be changed once the lib is out...

Reply:
yes once we deploy lib as public I will change this